README.md

```diff
@@ -1,16 +1,248 @@
-# Title
+# Methods in Ruby
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+- Describe routines and procedures for our code using methods
+- Define a method
+- Recognize how to call methods
 
 ## Introduction
 
-## SWBAT 1
+A method in Ruby is a _block_ of code that returns a value. With methods, we can
+organize our code into chunks of reusable functionality that can be easily used
+by other areas of their program.
 
-## SWBAT 2
+Ruby methods are very similar to functions in any other programming language.
+Ruby methods are used to bundle one or more repeatable statements into a single
+unit.
+
+In this lesson we'll introduce methods, distinguish them from data types, and
+cover how to create and execute them in your Ruby program.
+
+## Describe Routines and Procedures for Our Code Using Methods
```

Comment:
This LG was murky (per note above) and now I'm not sure how to sense if this is good.

After a few rereads, I think the LG is: "Demonstrate abstraction"
(and possibly also)

"Define DRY"